sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -229,6 +229,8 @@ class SQLQueryTestSuite extends QueryTest with SharedSQLContext {
     // List of SQL queries to run
```

Comment:
Shall you change line 227 together in this PR in order to remove the comment-only lines? It will remove 41 lines additionally.
```scala
-    val (comments, code) = input.split("\n").partition(_.startsWith("--"))
+    val (comments, code) = input.split("\n").partition(_.trim.startsWith("--"))
```